modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm

```diff
@@ -280,12 +280,51 @@
 	var/obj/item/organ/genital/breasts/mob_breasts = exposed_mob.getorganslot(ORGAN_SLOT_BREASTS)
 	enlargement_amount += enlarger_increase_step
 	if(enlargement_amount >= enlargement_threshold)
-		if(mob_breasts?.genital_size >= max_breast_size)
-			return
```

Comment:
Why have you got rid of the cap, exactly?

Reply:
I removed the cap because "beyond measurement" is never used, though it can be readded if it causes issues

Reply:
Unless I'm being blind, won't it just make them expand indefinitely?

Reply:
as long as the chemical's in them and continues increasing the enlargement_amount value, yeah. i don't believe it'd cause any issues, aside from it being significantly harder to make them smaller

Reply:
i'll readd a hard cap at the max breast size + 5, just so the beyond measurement text can be used - since it's unused in genitals.dm

Reply:
I mean, I feel like `max_breast_size` should be... the maximum breast size. Increase the cap if you really want, I suppose.